command-line/command-line-advanced.md

```diff
@@ -41,38 +45,6 @@ Teach people some handier tools for advanced file searching and processing
 - [Filenames and Pathnames in Shell: How to do it Correctly](https://www.dwheeler.com/essays/filenames-in-shell.html)
 - [An Awk Primer/Awk Command-Line Examples](https://en.wikibooks.org/wiki/An_Awk_Primer/Awk_Command-Line_Examples)
 - [I/O Redirection](http://wiki.bash-hackers.org/syntax/redirection)
-- [Video](https://www.youtube.com/watch?v=jxQzuha8TbU): About 3 minutes. Cover apropos command.
-- [Video](https://www.youtube.com/watch?v=CdWazJlfue8): About 2 minutes. Covers command pushd, popd commands.
-- [Video](https://www.youtube.com/watch?v=zcHGcIu_65k): About 6 minutes. Cover xargs command.
-- [Video](https://www.youtube.com/watch?v=NodKFuUocQY): About 22 minutes. Covers command aliases, whereis, service, passwd, df, du, wget, who,ping,shutdown commands
-    - 1:46 aliases command
-    - 7:32 whereis command
-    - 8:45 about service
-    - 10:38 service command
-    - 12:55 df command
-    - 13:40 du command
-    - 15:03 passwd command
-    - 16:15 wget command
-    - 17:38 who command
-    - 18:45 ping command
-    - 20:05 shutdown command
-- [Video](https://www.youtube.com/watch?v=nLa6jAbULe8): About 8 minutes. Covers command cut, &&, sort, uniq, wc, grep commands. 
-    - 0:59 && operator
-    - 3:34 cut command
-    - 4:28 sort command
-    - 4:59 uniq command
-    - 5:20 wc command
-    - 5:40 grep command
-
-- [Video](https://www.youtube.com/watch?v=yCTnihfbPCo): About 55 minutes. Covers command grep, sed, awk, tar, less, gzip commands.
-    - 1:25 grep command
```

Comment:
I removed this timings . Is it ok?

Reply:
Actually we asked a participant to add those, please put them back =)